packages/sdk/src/plt/Token.ts

```diff
@@ -198,6 +279,14 @@ class Token {
     ) {
         this.moduleState = Cbor.decode(info.state.moduleState, 'TokenModuleState');
     }
+
+    /**
+     * Returns a new Token instance with updated info and parsed module state.
+     */
+    public async updateToken(): Promise<Token> {
+        const updatedInfo = await this.grpc.getTokenInfo(this.info.id);
+        return new Token(this.grpc, updatedInfo);
+    }
```

Comment:
I would prefer if we mutated the existing token here, I don't see any reason for the user to keep outdated token info around. Can still a reference to the updated token for convenience 🙂 

The way to do this in js/ts would be like this:

```ts
class Test {
  private another: number;

  constructor(private prop: string) {
    this.another = Number(prop);
  }

  public get prop(): string {  return this.prop; }
  public get another(): number {  return this.another; }
}
```

Please remember to update the docstring for `updateToken` and mention that it mutates the internal state of the token.